### ISSUE TITLE
秒记

### ISSUE THREAD

**Issue by zhaipro** [zhaipro/vos#1]

**Comment by zhaipro** [comment]
1. https://github.com/foolwood/SiamMask
1. https://arxiv.org/pdf/1812.05050.pdf
1. https://tongtianta.site/paper/8327
1. https://youtube-vos.org/
1. https://competitions.codalab.org/competitions/20127#participate-get-data
1. https://drive.google.com/drive/folders/1BWzrCWyPEmBEKm0lOHe5KLuBuQxUSwqz
1. https://github.com/andr345/frtm-vos
1. https://zhuanlan.zhihu.com/p/76460186
1. https://blog.csdn.net/shenziheng1/article/details/88586880
1. https://blog.csdn.net/sgfmby1994/article/details/79865866
1. http://www.robots.ox.ac.uk/~luca/siamese-fc.html
1. https://www.zhihu.com/question/61607442
1. https://paperswithcode.com/task/video-object-segmentation/latest
2. https://arxiv.org/pdf/2007.08139v1.pdf
3. https://github.com/yuk6heo/IVOS-ATNet

**Comment by zhaipro** [comment]
1. https://github.com/keras-team/keras-applications/blob/master/keras_applications/resnet50.py#L142
1. https://keras.io/zh/applications/#resnet

**Comment by zhaipro** [comment]
response of a candidate window(RoW)
候选窗口
z 样本图
x 搜索图

**Comment by zhaipro** [comment]
最简单的一道题
输入图片和方格，输出mask

**Comment by zhaipro** [comment]
Siamese Net 孪生神经网络
`loss(y, v) = log(1 + exp(-yv))`
正样本 +1
负样本 -1

**Comment by zhaipro** [comment]
我觉得自己的熵小了点，噪音降低，信息增加。
https://new.qq.com/omn/20200107/20200107A082XI00.html

**Comment by zhaipro** [comment]
![im](https://user-images.githubusercontent.com/8620842/98909525-e9ee5000-24fc-11eb-91f5-864c5b0c127b.jpg)
![x_crop](https://user-images.githubusercontent.com/8620842/98909543-ef4b9a80-24fc-11eb-9a0d-61e9868f6a2f.jpg)
![z_crop](https://user-images.githubusercontent.com/8620842/98909553-f1155e00-24fc-11eb-9e03-b7a4b0fea05b.jpg)

**Comment by zhaipro** [comment]
255 127 17 是有意义的

**Comment by zhaipro** [comment]
算法跑通了，需要做优化了。

**Comment by zhaipro** [comment]
1. 数据增强
1. 加大视野
1. 优化损失
1. 不同的宽高比视角

**Comment by zhaipro** [comment]
不需要反例

**Comment by zhaipro** [comment]
Epoch 62/100
5814/5814 [=] - 937s 161ms/step - loss: 0.0712 - val_loss: 0.1089 - lr: 1.0000e-08

Epoch 66/100
5814/5814 [=] - 946s 163ms/step - loss: 0.7741 - reshape_4_loss: 0.0865 - conv2d_5_loss: 0.6876 - val_loss: 0.8334 - val_reshape_4_loss: 0.1076 - val_conv2d_5_loss: 0.7258 - lr: 1.0000e-09

Epoch 100/100
5814/5814 [=] - 915s 157ms/step - loss: 0.0676 - val_loss: 0.1497 - lr: 1.0000e-12

Epoch 95/100
830/830 [=] - 759s 915ms/step - loss: 0.2050 - val_loss: 0.2660 - lr: 1.0000e-06

Epoch 100/100
830/830 [=] - 763s 919ms/step - loss: 0.1453 - conv2d_20_loss: 0.0882 - conv2d_5_loss: 0.0571 - val_loss: 0.1593 - val_conv2d_20_loss: 0.0868 - val_conv2d_5_loss: 0.0724 - lr: 1.0000e-07

Epoch 100/100
830/830 [=] - 759s 914ms/step - loss: 0.4110 - conv2d_20_loss: 0.4063 - conv2d_5_loss: 0.0047 - val_loss: 0.5524 - val_conv2d_20_loss: 0.5278 - val_conv2d_5_loss: 0.0245 - lr: 1.0000e-10

20201225 Epoch 91/100
830/830 [=] - 348s 419ms/step - loss: 0.2701 - val_loss: 0.5358 - lr: 1.0000e-07

**Comment by zhaipro** [comment]
经验教训：

1. 不要过度的数据增强
1. [Why all bias of nn.Conv2d is False?](https://github.com/kuangliu/pytorch-cifar/issues/52)
1. 批量很重要

**Comment by zhaipro** [comment]
thr_high = 0.6
thr_low = 0.3

**Comment by zhaipro** [comment]
简单粗暴？
粗暴也要适可而止？

**Comment by zhaipro** [comment]
有绝望也有幸起

我明明知道起始帧物体的具体方格，却没告诉你。

**Comment by zhaipro** [comment]
我也需要深度学习啦…

**Comment by zhaipro** [comment]
我们需要模型能够识别未知目标

这就需要我们赋予现学的能力

**Comment by zhaipro** [comment]
针对视频逐帧处理，会使错误不断的叠加

**Comment by zhaipro** [comment]
anchor boxes
